client/.eslintrc.json

```diff
@@ -0,0 +1,62 @@
+{
```

Comment:
re: the eslint errors in CI, you can go ahead and add the tailwind config file to a .eslintignore file!

Reply:
ditto for the babel config file